scripts/includes/common.sh

```diff
@@ -0,0 +1,248 @@
+
+  if [ ! -d ${ROOT_USER_CLI_CONFIG_DIRECTORY} ]; then
+    sudo mkdir -p ${ROOT_USER_CLI_CONFIG_DIRECTORY}
+  fi
+
+  sudo sh -c "cat <<EOT > ${ROOT_USER_CLI_CONFIG_PATH}
+[credentials]
+username = ${USERNAME}
+password = ${PASSWORD}
+EOT"
+
+  # Write config for root user
+  if [ "${CURRENT_USER}" == "${ROOT_USER}" ]; then
+      return
+  fi
+
+  # Write config for current user (in case current user != root)
+  if [ ! -d ${CURRENT_USER_CLI_CONFIG_DIRECTORY} ]; then
+    sudo mkdir -p ${CURRENT_USER_CLI_CONFIG_DIRECTORY}
+  fi
+
+  sudo sh -c "cat <<EOT > ${CURRENT_USER_CLI_CONFIG_PATH}
+[credentials]
+username = ${USERNAME}
+password = ${PASSWORD}
+EOT"
+
+  # Fix the permissions
+  sudo chown -R ${CURRENT_USER}:${CURRENT_USER} ${CURRENT_USER_CLI_CONFIG_DIRECTORY}
+}
+
+
+generate_symmetric_crypto_key_for_datastore() {
+  DATASTORE_ENCRYPTION_KEYS_DIRECTORY="/etc/st2/keys"
+  DATASTORE_ENCRYPTION_KEY_PATH="${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}/datastore_key.json"
+
+  sudo mkdir -p ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
+  sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
+
+  # Make sure only st2 user can read the file
+  sudo usermod -a -G st2 st2
+  sudo chgrp st2 ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
+  sudo chmod o-r ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
+  sudo chgrp st2 ${DATASTORE_ENCRYPTION_KEY_PATH}
+  sudo chmod o-r ${DATASTORE_ENCRYPTION_KEY_PATH}
+
+  # set path to the key file in the config
+  sudo crudini --set /etc/st2/st2.conf keyvalue encryption_key_path ${DATASTORE_ENCRYPTION_KEY_PATH}
+
+  sudo st2ctl restart-component st2api
+}
+
+
+verify_st2() {
+  st2 --version
+  st2 -h
+
+  st2 auth $USERNAME -p $PASSWORD
+  # A shortcut to authenticate and export the token
+  export ST2_AUTH_TOKEN=$(st2 auth $USERNAME -p $PASSWORD -t)
+
+  # List the actions from a 'core' pack
+  st2 action list --pack=core
+
+  # Run a local shell command
+  st2 run core.local -- date -R
+
+  # See the execution results
+  st2 execution list
+
+  # Fire a remote comand via SSH (Requires passwordless SSH)
+  st2 run core.remote hosts='127.0.0.1' -- uname -a
+
+  # Install a pack
+  st2 pack install st2
+}
+
+
+ok_message() {
+  echo ""
+  echo ""
+  echo "███████╗████████╗██████╗      ██████╗ ██╗  ██╗";
+  echo "██╔════╝╚══██╔══╝╚════██╗    ██╔═══██╗██║ ██╔╝";
+  echo "███████╗   ██║    █████╔╝    ██║   ██║█████╔╝ ";
+  echo "╚════██║   ██║   ██╔═══╝     ██║   ██║██╔═██╗ ";
+  echo "███████║   ██║   ███████╗    ╚██████╔╝██║  ██╗";
+  echo "╚══════╝   ╚═╝   ╚══════╝     ╚═════╝ ╚═╝  ╚═╝";
+  echo ""
+  echo "  st2 is installed and ready to use."
+  echo ""
+  echo "Head to https://YOUR_HOST_IP/ to access the WebUI"
+  echo ""
+  echo "Don't forget to dive into our documentation! Here are some resources"
+  echo "for you:"
+  echo ""
+  echo "* Documentation  - https://docs.stackstorm.com"
+  echo "* Knowledge Base - https://stackstorm.reamaze.com"
```

Comment:
These are old articles, should not point users to reamaze or update them.

Reply:
Yep. I considered changing now, but ultimately decided to do in a subsequent PR.